extensions/common/util.h

```diff
@@ -17,11 +17,28 @@
 
 #include <string>
 
+#include "absl/strings/string_view.h"
+
 namespace Wasm {
 namespace Common {
 
+// None response flag.
+const std::string NONE = "-";
+
 // Parses an integer response flag into a readable short string.
 const std::string parseResponseFlag(uint64_t response_flag);
 
+// Used for converting sanctioned uses of std string_view (e.g. extensions) to
```

Comment:
Could you use source/common/common/stl_helpers.h ?

Reply:
We probably can't because of Wasm compilation method. We're trying to keep code minimal since Wasm is not as fast.

Reply:
Yeah we'd need to copy those two functions for Wasm module build. It is hard to introduce any Envoy source dep.